asset-tracking-location-replaying/.gitignore

```diff
@@ -0,0 +1,5 @@
+/node_modules/
+.env
+
+.ably_credentials.json
+.script_configuration.json
```

Comment:
There should probably be a new line at the end of this file

Reply:
https://github.com/ably/ably-asset-tracking-common/pull/25/commits/b5deb78c5f2fe9ecf1f78119e6ad369e967470d6